### ISSUE TITLE
[BUG] plugin doesen't recognize "min_char_count" parameter

### ISSUE THREAD

**Issue by CyberGuerro** [norberttak/XPanel#107]
As in the Title, it seems that plugin doesen't recognize the parameter "min_char_count".
Here an extract from config file:
```
[multi_display:id="RADIO_DISPLAY_ACTIVE_DOWN"]
line="on_select:SW_DOWN_COM1,dataref:sim/cockpit/radios/com1_freq_hz,dot_position:2"
line="on_select:SW_DOWN_COM2,dataref:sim/cockpit/radios/com2_freq_hz,dot_position:2"
line="on_select:SW_DOWN_NAV1,dataref:sim/cockpit/radios/nav1_freq_hz,dot_position:2"
line="on_select:SW_DOWN_NAV2,dataref:sim/cockpit/radios/nav2_freq_hz,dot_position:2"
line="on_select:SW_DOWN_ADF,dataref:sim/cockpit/radios/adf1_freq_hz"
line="on_select:SW_DOWN_DME,dataref:sim/cockpit/radios/nav1_dme_dist_m"
line="on_select:SW_DOWN_IDT,dataref:sim/cockpit/radios/transponder_code,min_char_count:4"
```
and error returned:
```
XPanel [4709] [ERROR]:parser: invalid syntax (section starts at line: 200): on_select:SW_DOWN_IDT,dataref:sim/cockpit/radios/transponder_code,min_char_count:4
XPanel [4709] [ERROR]:parser: error while processing key: line
XPanel [4714] [ERROR]:error parsing config file
XPanel [4714] [ERROR]:error during plugin init and start
```

**Comment by norberttak** [comment]
The correct parameter name is ```minimum_digit_number``` 
It's an error in my documentation what mentions the min_char_count. I need to fix it. Sorry for the misleading doc.

**Comment by norberttak** [comment]
PR for the documentation fix: #108

**Comment by CyberGuerro** [comment]
Ok, corrected and now workd as expected.
THX
